lib/types.ts

```diff
@@ -196,6 +196,9 @@ export function callVariadicCallback<T>(
 
 export type Callback<T> = CallbackNoError<T> | CallbackWithError<T>;
 
+/**
+ * Internal type, sent to the Raygun.com as part of MessageDetails
```

Comment:
```suggestion
 * Internal type, sent to the Raygun API as part of MessageDetails
```

Does this make sense to change?

Reply:
We could then even say `Raygun Crash Reporting API`